smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HttpResponseGenerator.kt

```diff
@@ -33,11 +33,17 @@ class HttpResponseGenerator(
             }
         }
 
+        httpResponseBindingErrorGenerator.renderServiceError(ctx)
         httpOperations.forEach {
-            httpResponseBindingErrorGenerator.render(ctx, it, unknownServiceErrorSymbol)
+            httpResponseBindingErrorGenerator.renderOperationError(ctx, it, unknownServiceErrorSymbol)
         }
 
-        val modeledErrors = httpOperations.flatMap { it.errors }.map { ctx.model.expectShape(it) as StructureShape }.toSet()
+        val modeledOperationErrors = httpOperations.flatMap { it.errors }.map { ctx.model.expectShape(it) as StructureShape }.toSet()
+        var modeledServiceErrors: MutableSet<StructureShape> = mutableSetOf()
+        for (serviceError in ctx.service.errors) {
+            modeledServiceErrors.add(ctx.model.expectShape(serviceError) as StructureShape)
+        }
+        val modeledErrors = modeledOperationErrors + modeledServiceErrors
```

Comment:
Spacing would help with readability here :) 
```
val modeledOperationErrors = httpOperations
    .flatMap { it.errors }
    .map { ctx.model.expectShape(it) as StructureShape }
    .toSet()
```

And for modeledServiceErrors we can achieve similar syntax to what you used in modeledOperationErrors with map
```
val modeledServiceErrors = ctx.service.errors
    .map { ctx.model.expectShape(it) as StructureShape }
    .toSet()
```

Reply:
Changed as suggested.